tests/codegen/intrinsics/nontemporal.rs

```diff
@@ -1,13 +1,28 @@
 //@ compile-flags: -O
+//@ compile-flags: --target aarch64-unknown-linux-gnu
+//@ needs-llvm-components: aarch64
```

Comment:
Question [REV1 (1/2)]: do we want to have revisions for:

- Each of the known "well-behaved" architectures (subject to availability of llvm components) where `!nontemporal` is emitted (i.e. `CHECK`)
- Examples of "naughty" architectures like x86_64 where `!nontemporal` is explicitly not emitted (i.e. `CHECK-NOT`)

So that we have both positive test cases where we emit `!nontemporal` and negative test cases where we resort to regular stores?

Reply:
I don't think it makes sense to repeat the list of well-behaved architectures here.

But yeah I can add a variant for x86 to ensure we do not emit the flag there, that's a good point.

Reply:
Yeah, that seems fine to me!